mkdocs.yml

```diff
@@ -120,6 +120,7 @@ nav:
         - Projects: tutorials/projects.md
         - Deployments: tutorials/deployments.md
         - Storage & Infrastructure: tutorials/storage.md
+        - Developing a New Worker Type: tutorials/developing-a-new-worker-type.md
```

Comment:
Yeah, I think the consensus was to have it in tutorials for now, but I also expect that it will move in the future as we produce more content like this.